src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/ECDsa/ECDsaTests.netcoreapp.cs

```diff
@@ -80,6 +80,7 @@ public void KeySizeProp()
         }
 
         [Theory, MemberData(nameof(TestNewCurves))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/51332", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```

Comment:
This should not be necessary anymore, fixed by AppleAppBuilder change.

Reply:
@filipnavara I just ran the test suite locally (based on 835f7c9a93f) and it seems to still fail with what I think are the same failures and messages.

Reply:
The latest CI run also saw consistent crashes of System.Security.Cryptography.Algorithms.Tests across iOSSimulator/tvOSSimulator and MacCatalyst, we should probably disable it for now too so we can get this PR in.

Reply:
@akoeplinger Are they the crashes from the AesGcm, AesCcm and ChaCha20Poly1305 tests that fail to load OpenSSL? I'd very much prefer to disable just the three test classes. https://github.com/dotnet/runtime/issues/52598

Reply:
The `AesGcm`, `AesCcm`, `ChaChaPoly1305` shouldn't even by trying to load the "real" implementation on non-macOS platforms since they are backed by OpenSSL. If these are indeed the cause then we should fix the project file so that we end up coming `AesGcm.NotSupported.cs` instead of `AesGcm.Unix.cs` for these platforms.

Reply:
@vcsjones Yes, it should totally be done. At the moment iOS/tvOS/MacCatalyst consume the same implemention as macOS. I didn't want to touch it since the code was still very much in flux for the past weeks due to introduction of ChaChaPoly.

Until that is done properly I would still prefer not to disable the whole test suite but just the three affected classes.

Reply:
@filipnavara https://gist.github.com/mdh1418/7ad77945032249078a8d438fe9321c25 I've been seeing problems with ChaCha20Poly1305 issues when running `System.Security.Cryptography.Algorithms.Tests` locally

Reply:
@mdh1418  The AesCcm, AesGcm and ChaCha20Poly1305 classes are supposed to throw PNSE on iOS-like platforms. The main reason why I didn't submit PR for it is that ChaChaPoly is new code and the same PR introduced `AesCcm.IsSupported` and `AesGcm.IsSupported` properties. The tests are now guarded by the new properties but the implementation still incorrectly returns `true` on iOS and/or attempts to load the OpenSSL library. In some cases it results in loading the native OpenSSL shim library which in turn `assert`s in native code because real OpenSSL is not available. The `assert` causes the whole test suite to crash.

Reply:
Obviously the correct fix is to fix all the `.IsSupported` methods to return `false` but since there are still open issues with the new code and @vcsjones attacks the same API surface I didn't want to interfere with the in-flight PRs.

We can do surgical workaround by disabling just the three test classes and re-enabling it once the dust settles and the implementation is fixed properly.

Reply:
Yup sounds good, but we'll still go ahead and merge this PR for now to get to a green state in runtime-staging :)